node/flatpak-node-generator.py

```diff
@@ -64,6 +64,8 @@
 
 GIT_URL_HOSTS = ['github.com', 'gitlab.com', 'bitbucket.com', 'bitbucket.org']
 
+NPM_MIRROR = 'https://cdn.jsdelivr.net/npm/'
```

Comment:
nit: might be worth using unpkg if possible? Since it's used elsewhere, for consistency

Reply:
Oh, I didn't notice a npm mirror is already used somewhere. Done.